### PR TITLE
Removes post endpoints and cleans up requirements

### PR DESCRIPTION
**Pull request by cgil**
Domain Reviewers: @mznco @TheBlueFool @staydomio/backend-reviewers

### CODE REVIEW

File: properties/app.py [hunk 1]
```diff
@@ -2,53 +2,65 @@
...
 
 
-@app.route('/properties', methods=['GET'])
 @app.route('/properties/', methods=['GET'])
 def get_all_properties():
-    return jsonify(properties=database.get_all_properties())
+    properties = database.get_all_properties()
+    return jsonify(
+        properties=properties,
+        count=len(properties),
```
Comment (cgil) [inline properties/app.py:26]: Hoping this counter makes it more obvious that the returned payload is a json dict.

File: properties/app.py [hunk 2]
```diff
@@ -2,53 +2,65 @@
 
 from flask import Flask
 from flask import jsonify
-from flask import request
+# from flask import request
 from flask_cors import CORS
 
+
 app = Flask(__name__)
+app.url_map.strict_slashes = False
```
Comment (cgil) [inline properties/app.py:10]: Allows `/owners/` as well as `/owners` as candidates keep tripping up over this.

**Review by willnevillain — APPROVED** [review]
Yeah this all seems fine.

File: properties/app.py (hunk 2 above)
Comment (willnevillain) [reply properties/app.py:10]: Very good, it's annoying.

**Pull request merged**